ios/Podfile

```diff
@@ -7,8 +7,8 @@ platform :ios, deployment_target
 ENV['COCOAPODS_DISABLE_STATS'] = 'true'
 
 project 'Runner', {
-'Debug' => :debug,
-	'Profile' => :release,
+	'Debug' => :debug,
```

Comment:
❓ Are these changes just leftovers?

Reply:
Nope, I think `profile` should be executed with debug flavour.